pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes Konvoy 1.8.4
+title: Release Notes Konvoy 1.8.4
+menuWeight: 40
+excerpt: View release-specific information for Konvoy 1.8.4
+beta: false
+enterprise: false
+---
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Konvoy&reg; version 1.8.4 was released on **
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.18.x  |
+| **Maximum**  | 1.20.x  |
+| **Default**  | 1.20.11  |
```

Comment:
```suggestion
| **Default**  | 1.20.13  |
```

Reply:
Hey @dlipovetsky! Thanks for your review. To my knowledge, a GA release is being pushed this Friday with more fixes related to Konvoy 1.8.4, hence the notes for that release. 

Feel free to reach out to me here or on Slack (@mwright) with any further questions.